src/Utility.js

```diff
@@ -92,6 +92,18 @@ const __class = lang.setObject('crm.Utility', lang.mixin({}, Utility, {
     }
     return id;
   },
+  trimText: function trimText(text = {}, wordCount = {}) {
+    const words = text.split(' ');
+    if (words.length > wordCount) {
+      const intermediate = words.slice(0, wordCount);
+      if (intermediate[wordCount - 1].slice(-1) === '.') {
+        intermediate[wordCount - 1] = intermediate[wordCount - 1].slice(0, -1);
+      }
+      const value = intermediate.join(' ') + ' ...';
```

Comment:
They can do multi-line. They are easier to read as the string grows. I do not find it harder to read.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/template_strings